app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -47,6 +50,43 @@ const MigrationsCompletedList = ({
               new Date(mostRecentRequest && mostRecentRequest.fulfilled_on)
             );
 
+            const archiveMigrationWarningText = (
+              <React.Fragment>
+                <p>
+                  {__('Are you sure you want to archive migration plan ')}
+                  <strong>{plan.name}</strong>?
+                </p>
+                {failed && (
+                  <p>
+                    {__(
+                      'This plan includes VMs that failed to migrate. If you archive the plan, you will not be able to retry the failed migrations'
```

Comment:
Missing period at the end

"If you archive the plan, you will not be able to retry the failed migrations."